api/views/submission.py

```diff
@@ -177,6 +179,38 @@ def get_operation_id(self, path, method):
         return super().get_operation_id(path, method)
 
 
+def get_access_conditions_url(training_type, training_acronym, partial_training_acronym):
+    if training_type == TrainingType.PHD.name:
+        return (
+            "https://uclouvain.be/en/study/inscriptions/doctorate-and-doctoral-training.html"
+            if get_language() == settings.LANGUAGE_CODE_EN
+            else "https://uclouvain.be/fr/etudier/inscriptions/conditions-doctorats.html"
+        )
+    else:
+        # Get the last year being published
+        today = timezone.now().today()
+        year = (
+            AcademicCalendar.objects.filter(
+                reference=AcademicCalendarTypes.ADMISSION_ACCESS_CONDITIONS_URL.name,
+                start_date__lte=today,
+            )
+            .order_by('-start_date')
+            .values_list('data_year__year', flat=True)
+            .first()
+        )
+
+        sigle = training_acronym
+        with suppress(ProgramTreeNotFoundException):  # pragma: no cover
+            # Try to get the acronym from the parent, if it exists
+            parent = ProgramTreeRepository.get_identite_parent_2M(partial_training_acronym, year)
+            sigle = EducationGroupYear.objects.get(
+                partial_acronym=parent.code,
+                academic_year__year=parent.year,
+            ).acronym
```

Comment:
Utilise plutot la commande : GetSigleFormationParenteQuery

Reply:
Je pensais le faire dans une autre PR si cela te convient (cf [ce ticket](https://jira.sgsi.ucl.ac.be/browse/OS-460)).

Reply:
Pas de soucis si ce n'est pas oublié :p